src/wiki/sites.py

```diff
@@ -0,0 +1,182 @@
+from django.apps import apps
+from django.utils.functional import LazyObject
+from django.utils.module_loading import import_string
+from wiki.compat import include, url
+from wiki.core.plugins import registry
+
+
+class WikiSite:
+
+    """
+    Wiki site configurator.
+
+    To customize, you can define your own subclass, either overriding
+    the view providers, or overriding the functions that collect
+    views and set update WikiConfig.default_site to the dotted import path
+    of your customized site.
+    """
+
+    def __init__(self, name='wiki'):
+        from wiki.conf import settings
+        from wiki.views import accounts, article, deleted_list
+
+        self.name = name
+
+        # root view
+        self.root_view_class = article.CreateRootView
+        self.root_missing_view_class = article.MissingRootView
+
+        # basic views
+        self.article_view_class = getattr(self, "article_view_class", article.ArticleView)
```

Comment:
I'm thinking about removing the `_class` part and require a view as in
`self.article_view = getattr(self, "article_view", article.ArticleView.as_view())`
Simpler, and less characters.

Reply:
Yes, since this is an entirely new API, I would think there is no problem in breaking how things were named in the old URL config class?

Reply:
Nobody use them, so there is no BC to preserve :smiley_cat: